### PR TITLE
Countdown Assignment

### PR DESCRIPTION
**Pull request by git-ebetts**
2 solutions for level 1 and one solution for level 2

### CODE REVIEW

File: countdown.js [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+// SOLUTION LEVEL 2
+
+function countdown(seconds) {
+    var intervalID = setInterval(printSeconds, 1000);
+    function printSeconds() {
+        console.log(seconds);
+        seconds --;
+        var timeoutID = setTimeout(function() {clearInterval(intervalID);}, [6000]);
+    }
+}
```
Comment (afeld) [inline countdown.js:40]: With this solution, what happens when you call `countdown()` with some number other than `5`? Does it still work as expected?

File: countdown.js [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+function countdown(seconds) {
+    var intervalID = setInterval(printSeconds, 1000);
+    function printSeconds() {
+        console.log(seconds);
+        seconds --;
+        if (seconds < 0) {
+            clearInterval(intervalID);
+        }
+    }
+}
```
Comment (afeld) [inline countdown.js:28]: This one is actually your best solution. Looks good!

File: countdown.js (hunk 1 above)
Comment (git-ebetts) [inline countdown.js:40]: Thanks for this question! I fixed the timeout and updated the file. Should I submit another pull request for this fix?

Comment (afeld) [inline countdown.js:40]: No need! Feel free to continue committing and pushing/syncing to this same one.
